bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/admin/TableAdapter.java

```diff
@@ -41,37 +44,34 @@
       ColumnDescriptorAdapter.INSTANCE;
   protected final BigtableInstanceName bigtableInstanceName;
 
-
-
   /**
    * <p>adapt.</p>
    *
-   * @param desc a {@link org.apache.hadoop.hbase.HTableDescriptor} object.
-   * @return a {@link com.google.bigtable.admin.v2.Table} object.
+   * This method adapts ColumnFamily to CreateTableRequest.
+   *
+   * @param  createTableRequest a {@link CreateTableRequest}
+   * @param desc a {@link HTableDescriptor} object.
    */
-  protected static Table adapt(HTableDescriptor desc) {
-    Map<String, ColumnFamily> columnFamilies = new HashMap<>();
+  @InternalApi
+  protected static void adapt(CreateTableRequest createTableRequest, HTableDescriptor desc) {
     for (HColumnDescriptor column : desc.getColumnFamilies()) {
       String columnName = column.getNameAsString();
-      ColumnFamily columnFamily = columnDescriptorAdapter.adapt(column);
-      columnFamilies.put(columnName, columnFamily);
+      createTableRequest.addFamily(columnName, buildGarbageCollectionRule(column));
     }
-    return Table.newBuilder().putAllColumnFamilies(columnFamilies).build();
   }
 
-  public static CreateTableRequest.Builder adapt(HTableDescriptor desc, byte[][] splitKeys) {
-    CreateTableRequest.Builder builder = CreateTableRequest.newBuilder();
-    builder.setTableId(desc.getTableName().getQualifierAsString());
-    builder.setTable(adapt(desc));
-    addSplitKeys(builder, splitKeys);
-    return builder;
+  @InternalApi
+  public static CreateTableRequest adapt(HTableDescriptor desc, byte[][] splitKeys) {
+    CreateTableRequest createTableRequest = CreateTableRequest.of(desc.getTableName().getNameAsString());
```

Comment:
Please call the variable `request` for short

Reply:
Would use shorter name here and for other occurrences as well.